packages/lodestar/src/chain/attestation.ts

```diff
@@ -69,6 +69,10 @@ export class AttestationProcessor implements IAttestationProcessor {
   }
 
   public async receiveBlock(signedBlock: SignedBeaconBlock): Promise<void> {
+    // process block's attestations
+    const attestations = signedBlock.message.body.attestations.valueOf() as Attestation[];
+    await Promise.all(attestations.map((a) => this.receiveAttestation(a)));
```

Comment:
I have to do `.valueOf() as Attestation[]` because `.map` is not available in `List<Attestation>`. Is it ok, any better way?

Reply:
This is good